qa/TL1_separate_executor/test_nofw.sh

```diff
@@ -9,17 +9,11 @@ do_once() {
 }
 
 test_body() {
-    start=`date +%s`
-    (sleep 10 && pkill -HUP ls && true) &
-    (ls /data/imagenet/train-jpeg > /dev/null && pkill -HUP sleep) &
-    wait
-    end=`date +%s`
-    runtime=$((end-start))
-    echo "Data access time: $runtime seconds"
-    if [ $runtime -gt 3 ]; then
-        echo "Data access time is greater than 3 seconds, skipping the test"
+    if [ $(stat  /mnt/data/ --format="%T" -f) != "ext2/ext3" ]; then
```

Comment:
so in this case, we are happy to not test this?

Reply:
Yes, this is just a part of all tests there so even if we skip some of it we should not miss much of our coverage.